### PR TITLE
Package field update method

### PR DESCRIPTION
**Pull request by JonoYang**
This PR adds a new method to the Package model to update fields. The idea is that we use these methods to update package fields, where it also updates the history as well.

### CODE REVIEW

File: packagedb/tests/test_models.py [hunk 1]
```diff
@@ -141,3 +143,191 @@ def test_packagedb_package_model_get_latest_version(self):
...
+                        'field': 'namespace',
+                        'old_value': '',
+                        'new_value': 'test'
+                    }
+                ]
+            }
+        }
+        history_entry = p1.history[0]
+        history_entry.pop('timestamp')
+        self.assertEqual(expected_history_entry, history_entry)
```
Comment (keshav-space) [inline packagedb/tests/test_models.py:170]: This one is not needed, right?

**Review by keshav-space — COMMENTED** [review]
Thanks @JonoYang
I was wondering, do we really need both `update_field` and `update_fields`? Isn't `update_fields` sufficient?

File: packagedb/tests/test_models.py (hunk 1 above)
Comment (JonoYang) [reply packagedb/tests/test_models.py]: Yes, I left this in by mistake. I've removed it.

**Comment by JonoYang** [comment]
@keshav-space That's true. I have the two because I wasn't sure what the best way to have the method arguments for updating a single field or multiple fields. I've removed `update_field` and `_update_field` and just left `update_fields`.

**Pull request merged**